docs/proposed/http-introducer.rst

```diff
@@ -0,0 +1,153 @@
+-------------------------
+
+Setup
+~~~~~
+
+You want to offer a collection of storage servers as a Tahoe-LAFS storage grid.
+
+The ``http-introducer`` tool is a stand-alone server which implements basic publish/subscribe functionality.
+
+The first step is to create the introducer's persistent state::
+
+  http-introducer create --config <path> --listen-endpoint <endpoint> [--certificate-path <path> --private-key-path <path>]
+
+This is an error if ``<path>`` exists already.
+If it does not then a new HTTP Introducer configuration is written there.
+If ``<path>`` is ``-`` then the configuration is written to stdout.
+
+``--listen-endpoint`` may be used repeatedly to listen on multiple addresses.
+Whatever endpoint type is chosen,
+``http-introducer`` will *always* _automatically_ negotiate TLS over it.
+
+If given,
+the files given by ``--certificate-path`` and ``--private-key-path`` have their contents read and added to the configuration state.
+After the execution of this command,
+these files are not read again.
+If not given,
+a new private key and self-signed certificate is generated and used.
+
+This is the HTTP Introducer's state and is required by future commands.
+It is the operator's responsibility to persist this state.
+
+The next step is to start the long-running HTTP Introducer process.
+As long as this is running,
+storage nodes will be able to publish announcements and clients will be able to subscribe to them.
+
+::
+   http-introducer run --config <path>
+
+``<path>`` should have previously been created by ``http-introducer create``.
+If ``<path>`` is ``-`` then the configuration is read from stdin.
+
+
+Enrolling Servers / Inviting Clients
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The configuration to make a storage server or a client node use an HTTP Introducer is the same.
+They each require the *introducer fURL*.
+To retrieve this value,
+run this command::
+
+  http-introducer get-introducer-furl --config <path>
+
+``<path>`` is handled here as elsewhere.
+The output is the *introducer fURL* which should be made available to a client node and referenced from that client's configuration.
+The configuration should be treated as secret because it includes secrets that allow access to the grid.
+The configuration is used in the process described by `Use As A Client`_ .
+
+Example Scenario
+----------------
+
+Alice operates an HTTP introducer.
+Bob operates a storage server.
+Carol operates a client node.
+
+::
+
+   [alice@aaa:~]$ CFG=myintroducer.json
+   [alice@aaa:~]$ http-introducer create --config $CFG --listen-endpoint tcp:12345
+   [alice@aaa:~]$ http-introducer get-introducer-furl --config $CFG
+   xxx://zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz@somewhere:12345/pqpqpqpqpqpqpqpqpqpqpqpqpqpqpqpq
+   [alice@aaa:~]$ daemonize http-introducer run --config $CFG
+   [alice@aaa:~]$
+
+::
+
+   [bob@bbb:~]$ cat >> storage-node/tahoe.cfg
+   [client]
+   http-introducer-path = storage-node/private/alicegrid.json
```

Comment:
Earlier the docs said this would be `private/alicegrid.json` (relative to node path).